src/test/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
```suggestion
public interface Drawable {
```
don't use verbs for class/interface names